src/api-reference/request/v4.endpoints.request-resources.markdown

```diff
@@ -571,7 +571,7 @@ PUT {datacenter}/travelrequest/v4/requests/{requestUuid}
 Name|Type|Format|Description
 ---|---|---|---
 `requestUuid`|`string`|-|**Required** The unique identifier of the Request.
-`userId`|`string`|-|The unique identifier of the user performing the update. **Required** when connecting with a Company token. If empty a 400, `missingRequiredParam` error code will be displayed.
+`userId`|`string`|-|The unique identifier of the user performing the update. Optionnal when connecting with a Company token.
```

Comment:
Optional ?

Reply:
We could add that it is automatically provided and will be ignored with a user token.

Reply:
add the line "Only used when connecting with a Company token, optional." - @laurentlhermine to confirm phrasing

Reply:
Suggesting the following :

`The unique identifier of the user performing the update. Optional. Will be taken into account only if calling with a Company token. If not provided the update will be performed as "Concur System".`